src/input/plugins/CurlInputPlugin.cxx

```diff
@@ -163,6 +209,9 @@ CurlInputStream::DoResume()
 {
 	assert(GetEventLoop().IsInside());
 
+	FmtDebug(curl_domain, "Resuming: capacity = {}, space = {}",
+		 GetBufferCapacity(), GetBufferSpace());
```

Comment:
Is logging such a thing really use full? This happens all the time, and is part of the normal workflow.